tests/v2/validation/provisioning/hosted/aks/hosted_provisioning_test.go

```diff
@@ -64,52 +65,34 @@ func (h *HostedAKSClusterProvisioningTestSuite) SetupSuite() {
 
 func (h *HostedAKSClusterProvisioningTestSuite) TestProvisioningHostedAKS() {
 	tests := []struct {
-		name            string
-		client          *rancher.Client
-		clusterName     string
-		cloudCredential *cloudcredentials.CloudCredential
+		name        string
+		client      *rancher.Client
+		clusterName string
```

Comment:
What is clusterName used for?

Reply:
That shouldn't be there, I'll update the code to remove that since I can declare it later.